src/components/ui/Chat/MessageAttachment/index.tsx

```diff
@@ -14,6 +14,14 @@ export interface MessageAttachmentProps
   name: string;
   /** The file URL string to download attachment. */
   downloadUrl: string;
+  /** Determines whether render image of the attachment. */
+  renderImg?: boolean;
+  /** The URL of image. */
+  imgUrl?: string;
+  /** The height of image. */
+  imgHeight?: number;
```

Comment:
If we are asking for a number here, I think we should specify the unit. Alternatively, we could use styled components for this and pass a string here. Either way.